Automattic/woocommerce-payments
# Add bundle size check to PR's

Fixes #4619

#### Changes proposed in this Pull Request

This PR adds a GH action for bundle size reporting. This is helpful to detect any unexpected size bump caused by the PR.

Here's a screenshot showing the size changes in a draft PR:

![Image 2022-08-25 at 2 30 37 PM](https://user-images.githubusercontent.com/6534920/186642971-f9fe6157-fb27-4dcc-a0d9-8117a70b7603.jpg)


#### Testing instructions

1. Clone the `add/4619-add-bundle-size-check` branch in your local machine.
2. Create a new branch on top of that.
3. Delete/add some code from some file(s).
4. Push that new branch.
5. Create a draft PR.
6. The size change report should be visible once the action completes.

-------------------


**Post merge**
